tests/crypto/test_cases/CMakeLists.txt

```diff
@@ -41,13 +42,16 @@ zephyr_sources_ifdef(CONFIG_MBEDTLS_ECDSA_C         test_vectors_ecdsa_random.c)
 zephyr_sources_ifdef(CONFIG_MBEDTLS_ECDSA_C         test_vectors_ecdsa_sign.c)
 zephyr_sources_ifdef(CONFIG_MBEDTLS_ECDSA_C         test_vectors_ecdsa_verify.c)
 zephyr_sources_ifdef(CONFIG_MBEDTLS_ECJPAKE_C       test_vectors_ecjpake.c)
+if(NOT CONFIG_CRYPTO_SKIP_HASH_TESTS)
```

Comment:
Please indent the lines between this `if()` and the following `endif()`

Reply:
Indented